easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-0.9.0-gompi-2019b.eb

```diff
@@ -0,0 +1,87 @@
+easyblock = 'CMakeMakeCp'
+
+name = 'RAxML-NG'
+version = '0.9.0'
+
+local_terrap_commit = '6dcb061ceb6ed41a7ebe1a8af1f3354e0126125c'
+local_pllmod_commit = '840a19525ed133ad243afb23124911b849d4685d'
+local_libpll_commit = '6874327c9ca9544e27b426c0e7205c8b6f4ed297'
+
+homepage = "https://github.com/amkozlov/raxml-ng"
+description = """RAxML-NG is a phylogenetic tree inference tool which
+uses maximum-likelihood (ML) optimality criterion. Its search heuristic
+ is based on iteratively performing a series of Subtree Pruning and Regrafting
+ (SPR) moves, which allows to quickly navigate to the best-known ML tree."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/amkozlov/raxml-ng/archive/'],
+        'filename': '%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/amkozlov/terraphast-one/archive/'],
+        'download_filename': local_terrap_commit + '.zip',
+        'filename': 'terraphist-one.zip',
+    },
+    {
+        'source_urls': ['https://github.com/ddarriba/pll-modules/archive/'],
+        'download_filename': local_pllmod_commit + '.zip',
+        'filename': 'pll-modules.zip',
+    },
+    {
+        'source_urls': ['https://github.com/xflouris/libpll-2/archive/'],
+        'download_filename': local_libpll_commit + '.zip',
+        'filename': 'libpll-2.zip',
+    },
+]
+
+checksums = [
+    '43b95ee1bddae0daee84644e9ee760a77f28bb16e2071e95cca79a30f39373e5',  # 0.9.0.tar.gz
+    'eef5e2f365619da3bb1679f07920b29dfcd5202191fdf37b9f8fec4a52c1286b',  # terraphist-one.zip
+    '486f55cc2b4a63ddba8d00db79afcbf03f2499e35687eef56e4730f23d96b90a',  # pll-modules.zip
+    '6c90af957d4b06b88ef6568f6ba53ac3507285d042dc9ebc95f9837a7077d044',  # libpll-2.zip
+]
+
+# copy dependencies into place
+preconfigopts = "cp -a %%(builddir)s/terraphast-one-%s" % local_terrap_commit
+preconfigopts += "/* %(builddir)s/raxml-ng-%(version)s/libs/terraphast/ && "
```

Comment:
```suggestion
preconfigopts = "cp -a %%(builddir)s/terraphast-one-%s/* " % local_terrap_commit
preconfigopts += "%(builddir)s/raxml-ng-%(version)s/libs/terraphast/ && "
```

Reply:
hm, you reverted these changes when you updated the PR (I guess that's the downside of using github code suggestions...), you'll need to incorporate them in your local file

Reply:
almost done :) please note

> hm, you reverted these changes when you updated the PR (I guess that's the downside of using github code suggestions...), you'll need to incorporate them in your local file

almost done! :) please note the above though (the changes we made to preconfigopts via github suggestions got reverted because they are not in your local eb file)

Reply:
F@#k. Sorry about that. My mistake.

I will update the pr again with the changes commited. But before doing that I have a question. I don't see any problem with the preconfig opts of terraphist-one and pll-modules. 

But the preconfig opts of libpll that you proposed will make the line wider than 80 characters. I think that doesn't comply with the pep8 style and the automatic tests fail because of that. Should I keep it in 3 lines of less than 80 or should I apply your proposal?

Reply:
in EB the limit is 120 lines, so it should be ok (you can check yourself with `eb --check-contrib` before updating the PR)